sources/platform/tutorial/workflow/sending-status-to-scm.md

```diff
@@ -38,6 +38,7 @@ resources:
       sourceName: <repoName>                    
       branch: <branchName>                      
       buildOnPullRequest: true     #specify true to trigger the job for pull requests to the git repository
+      buildOnPullRequestClose: true #specify true to trigger the runSh job for closed pull requests to the git repository
```

Comment:
lets leave it out of this example. it doesnt help with the "sending status to scm" scenario